perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/FacilitiesManagerMethod.java

```diff
@@ -94,6 +94,23 @@ public Facility call(ApiCaller ac, Deserializer parms) throws PerunException {
 	},
 
 	/*#
+	 * Returns all facilities that have set the attribute 'attributeName' with the value 'attributeValue'.
+	 * Searching only def and opt attributes. Large attributes are not supported.
+	 *
+	 * @param attributeName String
+	 * @param attributeValue String
+	 * @return Facilities Found facilities
```

Comment:
It should return: ``List<Facility>`` not ``Facilities`` - javadoc is parsed and it would fail.